test/test_subvariable.jl

```diff
@@ -73,7 +56,7 @@ ncvar_view = view(ncvar,1:3,1:4)
 ncvar_view.attrib["foo"] = "bar"
 @test ncvar_view.attrib["foo"] == "bar"
 @test ncvar.attrib["foo"] == "bar"
-@test SubVariable(ncvar)[:,:] == data
```

Comment:
There is not a `SubVariable` constructor like this any more. I could add one again for consistency if needed.